lib/apq/document_provider.ex

```diff
@@ -128,6 +133,12 @@ defmodule Apq.DocumentProvider do
         {:halt, %{request | document: {:apq_hash_format_error, nil}}}
       end
 
+      defp format_params(%{"extensions" => extensions} = params) when is_binary(extensions) do
+        Map.put(params, "extensions", Jason.decode!(extensions))
```

Comment:
Jason is just a test dependency in this library, and Absinthe allows you to use whichever codec you configure, so we need to make sure we use the configured codec, since Jason may not be available.(Thanks for putting this together by the way!)

Reply:
Here is where absinthe_plug uses the configured json codec https://github.com/absinthe-graphql/absinthe_plug/blob/c33d44cfd2c7db9bc3ad1feedc49571f6d019efd/lib/absinthe/plug/request.ex#L117